CHANGES.rst

```diff
@@ -304,6 +304,9 @@ Features added
 
 Bugs fixed
 ----------
+* #11433: The ``linkcheck_allow_unauthorized`` configuration option (default
+  value ``true``) has been added to allow treatment of HTTP 401 (unauthorized)
+  server responses as broken hyperlinks.
```

Comment:
I've a pending commit to clarify this wording to:

```
* #11433: The ``linkcheck_allow_unauthorized`` configuration option (default
  value ``true``) has been added.  Set this option to ``false`` to handle
  HTTP 401 (unauthorized) server responses as broken hyperlinks.
```

(I don't like pushing multiple commits while continuous integration is pending, when possible)

Reply:
...and I need to update the Sphinx docs to document this option.  I'll do that now.